### ISSUE TITLE
mc_lambda not passed to Tracker from strongSORT

### ISSUE THREAD

**Issue by lachlan-nicholson** [mikel-brostrom/Yolov5_StrongSORT_OSNet#452]
### Bug

The appearance to motion weight factor "mc_lambda" and exponential average parameter "ema_alpha" are not passed from StrongSORT to Tracker (strong_sort.py line 51). 

Additionally, Tracker class has two definitions of this weighting factor, mc_lambda and _lambda, where _lambda is used later such that 1.0 = only motion cost. mc_lambda is unused.

Finally, the config (strong_sort.yaml) describes mc_lambda as appearance cost (1-mc_lambda). 

I have fixed this locally by removing all references to _lambda, and correctly defining mc_lambda where 1.0 is full appearance.

**Comment by mikel-brostrom** [comment]
Yes, I have not cleaned everything up after the DeepSORT --> StrongSORT upgrade. I will try to take care of this, this week, thank you for the reminder :smile:

**Comment by q671383789** [comment]
Hey, @lachlan-nicholson  can u please specify how to fix this locally?

**Comment by lachlan-nicholson** [comment]
No dramas. Just thought I would mention in case others are unaware. 

@q671383789 I can submit a PR tomorrow if it helps you

**Comment by q671383789** [comment]
@lachlan-nicholson  thx for ur kindness, but I'm going to debug this myself which helps me go through the whole process .

**Comment by lachlan-nicholson** [comment]
@q671383789 if you're still looking for a description of my changes:

* tracker.py: I removed _lambda completely, and replaced it with mc_lambda. This means in the cost_matrix equation on line 130 I swapped _lambda -> (1-self.mc_lambda). 
* strong_sort.py: I correctly passed ema_alpha/mc_lambda to the tracker initialisation (line 51)
* strong_sort.yaml: I change the comment about MC_lambda to define appearance cost as (MC_LAMBDA) and motion cost (1-MC_LAMBDA).
